torch/quantization/fx/quantize.py

```diff
@@ -52,6 +52,18 @@ def quantize_node(node, activation_post_process):
 def quantize(quantizer, node):
     quantize_node(node, quantizer.activation_post_process_map[node.name])
 
+# returns a function that can get a new attribute name for module with given prefix
```

Comment:
can we add a real example to the docblock?  It would make the code easier to read, someone can just use it without reading every single line.